src/google/protobuf/util/internal/datapiece.cc

```diff
@@ -305,6 +306,9 @@ StatusOr<int> DataPiece::ToEnum(const google::protobuf::Enum* enum_type,
       value = FindEnumValueByNameWithoutUnderscoreOrNull(enum_type, enum_name);
       if (value != NULL) return value->number();
     }
+
+    // If ignore_unknown_enum_values is true an unknown enum value is treated as 0
```

Comment:
Please update the comment. It's treated as the default.

Reply:
I updated the commend and squashed my commits into one commit